index.html

```diff
@@ -561,16 +634,7 @@ <h3>
           </li>
         </ul>
```

Comment:
I personally would like to pull the list of configs given the adjustments, but can see reasoning to leave it in.